feat: added slot binds to the base form schema (#194)
vuelidate/formvuelatte: docs/3.x/src/guide/veevalidate.md

To use the plugin, import and pass it to the `SchemaFormFactory`. This creates a `SchemaForm` component with validation capabilities.

```js

> import { SchemaFormFactory } from "formvuelate";

Reviewer: Did you switch into double quotes and colons on purpose? We are using singles and no colons everywhere else on the docs, so if you don't mind let's keep it consistent :)

Reviewer: Looks like your linting made a ton of changes throughout the whole file XD

Author: Yea I don't know what happened there, will fix it.

```html
<template>
  <SchemaForm :schema="schema" :initial-touched="initialTouched">
    <template #afterForm>
      <button>Submit</button>
    </template>
  </SchemaForm>
</template>

<script>
  export default {
    setup() {
      const schema = ref([
        // schema...
      ]);

      const formData = ref({});
      useSchemaForm(formData);

      const initialTouched = {
        email: true,
        password: false
      };

      return {
        formData,
        schema,
        initialErrors,
        initialTouched
      };
    }
  };
</script>
```

> ## SchemaForm slot props

Reviewer: We'll need to add a pill here with the version of FVL and the plugin that supports it or clarify it somewhere

Author: Added a warning for both next minor releases.

> You can access the form-level validation state by using either `afterForm` or `beforeForm` slot prop named `validation` on the `SchemaForm` component.

Reviewer: Let's add a link here to `#slots`

      formData,
      schema,
      initialErrors,
      initialTouched
    }
  }
}
</script>
```

## SchemaForm slot props

::: warning Important

> This feature is only available in FormVuelate <Badge text="3.2" type="warning" vertical="middle" /> and @formvuelate/plugin-vee-validate <Badge text="2.3" type="warning" vertical="middle" />
> :::

Reviewer: ```suggestion
This feature is available starting on FormVuelate <Badge text="3.2" type="warning" vertical="middle" /> and @formvuelate/plugin-vee-validate <Badge text="2.3" type="warning" vertical="middle" />
:::
```

  </SchemaForm>
</template>
```

The following are a few common examples for behaviors implemented with the form-level validation state.

**Disable buttons until all fields are valid**

```html
<SchemaForm :schema="schema">
  <template #afterForm="{ validation }">
    <button :disabled="!validation.meta.valid">Submit</button>
  </template>
</SchemaForm>

> ```

Reviewer: Yeah there's some a11y considerations here, I don't want FVL docs to become a11y docs, but I also don't want to promote bad practices. We can keep the example if:

- Let's switch `:disabled` for `:aria-disabled`
- Add a subnote `Learn more about accessible disabling of form buttons` and link it to: https://css-tricks.com/making-disabled-buttons-more-inclusive/

**Display spinner or loading state when the form is submitting**

> ```vue

Reviewer: Did you mean html?

Author: 🤦‍♂️
